AT_CSC/atdome_setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Source this file when starting the container to set it up
+
+echo "#"
+echo "# Loading LSST Stack"
+. /opt/lsst/software/stack/loadLSST.bash
+setup lsst_distrib
+echo "#"
+echo "# Loading sal environment"
+. /home/saluser/repos/ts_sal/setup.env
+echo "#"
+echo "# Setting ATDome Simulator"
+
+cd ~/repos/ts_config_attcs/
+git fetch --all
+git checkout tickets/DM-22531
```

Comment:
has this been merged or do we still want to use this branch? Once you merge this and delete this branch won't it break the file?

Reply:
Good catch! let me get the config merged so I can fix this!

Reply:
Done! I'm using the new release tag. I'll create a ticket to re-do the deployment of the summit components next week with clean environment.